apps/els_lsp/src/els_completion_provider.erl

```diff
@@ -538,6 +549,139 @@ exported_definitions(Module, POIKind, ExportFormat) ->
       []
   end.
 
+%%==============================================================================
+%% Clauses
+%%==============================================================================
+
+-spec clause(els_dt_document:item(), line(), column()) -> [map()].
+clause(Document, Line, Column) ->
+  try
+  case els_dt_document:pois(Document, [function, function_clause]) of
```

Comment:
We should be able to only keep the try and remove the case?

Reply:
Yes, let me clean this up